nucliadb/src/nucliadb/search/search/predict_proxy.py

```diff
@@ -174,18 +175,17 @@ async def chat_streaming_generator(
     origin: str,
     user_query: str,
     is_json: bool,
+    metrics: AskMetrics,
 ):
     first = True
     first_reasoning = True
     status_code = AnswerStatusCode.ERROR.value
     text_answer = ""
     text_reasoning = ""
     json_object = None
-    metrics = AskMetrics()
     with metrics.time(PREDICT_ANSWER_METRIC):
         async for chunk in predict_response.content:
             yield chunk
-
             if is_json:
```

Comment:
can you change the name of this variable to be `is_ndjson_stream` ? Otherwise it can be a bit confusing witht the json generative response.